common/models/role.js

```diff
@@ -171,13 +171,6 @@ module.exports = function(Role) {
   Role.isOwner = function isOwner(modelClass, modelId, userId, callback) {
     assert(modelClass, 'Model class is required');
     debug('isOwner(): %s %s userId: %s', modelClass && modelClass.modelName, modelId, userId);
-    // No userId is present
-    if (!userId) {
```

Comment:
No that's the purpose of this PR : We want to be able to resolve multiple owners so if my two `blongsTo` relations are based on (for example) `requesterId` and `senderId` why would you want to DENY in case of no `userId` field ?